modules/core/src/index.js

```diff
@@ -79,8 +79,9 @@ export {default as log} from './utils/log';
 import {flattenVertices, fillArray} from './utils/flatten'; // Export? move to luma.gl or math.gl?
 
 export {createIterable} from './utils/iterable-utils';
-import {default as Tesselator} from './utils/tesselator'; // Export? move to luma.gl or math.gl?
+import Tesselator from './utils/tesselator'; // Export? move to luma.gl or math.gl?
```

Comment:
Are we actually revisiting these `TODO`s

Reply:
Out of scope for this PR